pkgs/by-name/tr/trealla/package.nix

```diff
@@ -1,38 +1,33 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, libffi
-, openssl
-, readline
-, valgrind
-, xxd
-, gitUpdater
-, checkLeaks ? false
-, enableFFI ? true
-, enableSSL ? true
-, enableThreads ? true
-, lineEditingLibrary ? "isocline"
+{
+  lib,
+  fetchFromGitHub,
+  gitUpdater,
+  libffi,
+  openssl,
+  readline,
+  stdenv,
+  valgrind,
+  xxd,
+  # Boolean flags
+  checkLeaks ? false,
```

Comment:
a comment about why you might not want to enable this test would be good. i'm assuming it's pretty taxing?

Reply:
Yes, and it is mostly for developers.